apis/workstations/v1alpha1/workstationcluster_types.go

```diff
@@ -56,12 +56,12 @@ type WorkstationClusterSpec struct {
 
 	// Immutable. Reference to the Compute Engine network in which instances associated
 	//  with this workstation cluster will be created.
-	NetworkRef *refs.ComputeNetworkRef `json:"networkRef,omitempty"`
+	NetworkRef refs.ComputeNetworkRef `json:"networkRef"`
```

Comment:
Nit: I suggest using the `// +required` option to make this explicit

Reply:
fixed!